src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const currentObject of actions) {
+    switch (currentObject.type) {
+      case 'addProperties':
+        Object.assign(state, currentObject.extraData);
+        break;
+      case 'removeProperties':
```

Comment:
add spaces between cases
```suggestion
      case 'addProperties':
        Object.assign(state, currentObject.extraData);
        break;

      case 'removeProperties':
```